rollup.config.js

```diff
@@ -47,7 +50,9 @@ const config = (args) => {
             },
             external: id => Object.keys(globals).some(key => id == key),
             plugins: [
-                resolveNode(),
+                resolveNode({
+                    mainFields: ['main'],
+                }),
```

Comment:
Tell the build to use `CJS` folder instead of `ESM`.

`Source v3.2` added support for `ESM` and `CJS`, and rollup defaults to using `ESM` if available. 
To avoid build errors (which could be false positives), we specify that we only want to use CJS

https://github.com/rollup/plugins/tree/master/packages/node-resolve#mainfields